test/utils/image/manifest.go

```diff
@@ -234,7 +234,7 @@ func initImageConfigs() map[int]Config {
 	configs[Perl] = Config{dockerLibraryRegistry, "perl", "5.26"}
```

Comment:
see if no one else using GBRedisSlaveImage then can we just remove an entry from here as well?

Reply:
`grep -r GBRedis test/` doesn´t return anything, seem is safe to remove it